GA01SUMO/GA01SUMO_D00002_ingest.csv

```diff
@@ -1,50 +1,50 @@
+mi.dataset.driver.phsen_abcdef.imodem.phsen_abcdef_imodem_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/imm/phsen2/phsen*.DAT,GA01SUMO-RII11-02-PHSENE042,telemetered,Available,
+mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/irid2shore/cpm_status*.txt,GA01SUMO-SBC11-00-CPMENG000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/syslog/*syslog.log,GA01SUMO-SBD11-00-DCLENG000,telemetered,Available,
+#mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/mopak/*.mopak.log,GA01SUMO-SBD11-01-MOPAK0000,telemetered,Not Expected,No Mopak Data
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/hyd*/*hyd*.log,GA01SUMO-SBD11-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/dosta1/*.dosta1.log,GA01SUMO-SBD11-04-DOSTAD000,telemetered,Available,
+mi.dataset.driver.spkir_abj.dcl.spkir_abj_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/spkir1/*.spkir1.log,GA01SUMO-SBD11-05-SPKIRB000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/metbk1/*.metbk*.log,GA01SUMO-SBD11-06-METBKA000,telemetered,Available,
+mi.dataset.driver.nutnr_b.dcl_full.nutnr_b_dcl_full_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl11/nutnr1/*.nutnr*.log,GA01SUMO-SBD11-08-NUTNRB000,telemetered,Available,
+mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/syslog/*syslog.log,GA01SUMO-SBD12-00-DCLENG000,telemetered,Available,
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/optaa1/*.optaa*.log,GA01SUMO-SBD12-01-OPTAAD000,telemetered,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/flort1/*.flort*.log,GA01SUMO-SBD12-02-FLORTD000,telemetered,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/hyd*/*hyd*.log,GA01SUMO-SBD12-03-HYDGN0000,telemetered,Available,
+mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/pco2a/*.pco2a.log,GA01SUMO-SBD12-04-PCO2AA000,telemetered,Available,
+mi.dataset.driver.wavss_a.dcl.wavss_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/wavss/*.wavss.log,GA01SUMO-SBD12-05-WAVSSA000,telemetered,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_telemetered_driver,/omc_data/whoi/OMC/GA01SUMO/D00002/cg_data/dcl12/metbk2/*.metbk*.log,GA01SUMO-SBD12-06-METBKA000,telemetered,Available,
```

Comment:
I don't believe I made any changes to this file... not sure why it's showing up as changed. Possibly opening and saving it before I closed made Git think it was all changed?